src/pages/experiments/[experimentId]/plots-and-tables/marker-heatmap/index.jsx

```diff
@@ -7,7 +7,7 @@ import {
   Select,
   Radio,
 } from 'antd';
-import _ from 'lodash';
+import _, { update } from 'lodash';
```

Comment:
When using lodash usually we import only the _ and then do _.update or whatever you want to use